docs/talks/intro-api/talk.Rmd

```diff
@@ -0,0 +1,349 @@
+
+```{r, fig.width=7, fig.height=5}
+hosp = covidcast_signal(data_source = "hospital-admissions", 
+                        signal = "smoothed_adj_covid19",
+                        start_day = "2020-03-01", geo_type = "state", 
+                        geo_values = c("pa", "ny", "tx"))
+
+plot(hosp, plot_type = "line", 
+     title = "% of hospital admissions due to COVID-19")
+```
+
+# Example: Cases 
+
+What does the current COVID-19 incident case rate look like, nationwide?
+
+```{r, fig.width=9, fig.height=7}
+cases = covidcast_signal(data_source = "jhu-csse", 
+                         signal = "confirmed_7dav_incidence_prop",
+                         start_day = "2020-08-30")
+
+plot(cases, title = "Daily new COVID-19 cases per 100,000 people")
+```
+
+# Example: Cases (Cont.)
+
+What does the current COVID-19 cumulative case rate look like, nationwide?
+
+```{r, fig.width=9, fig.height=7}
+cases = covidcast_signal(data_source = "jhu-csse", 
+                         signal = "confirmed_cumulative_prop",
+                         start_day = "2020-08-30")
+
+plot(cases, title = "Cumulative COVID-19 cases per 100,000 people", 
+     choro_params = list(legend_n = 6))
+```
+
+# Example: Cases (Cont.)
+
+Where is the current COVID-19 cumulative case rate greater than 2%?
+
+```{r, fig.width=9, fig.height=7}
+plot(cases, choro_col = c("#D3D3D3", "#FFC0CB"), 
+     title = "Cumulative COVID-19 cases per 100,000 people",
+     choro_params = list(breaks = c(0, 2000), legend_width = 5))
+```
+
+# Example: Doctor's Visits
+
+How do some major cities compare in terms of doctor's visits due to COVID-like illness?
+
+```{r, fig.width=7, fig.height=5}
+dv = covidcast_signal(data_source = "doctor-visits", 
+                      signal = "smoothed_adj_cli", 
+                      start_day = "2020-03-01", geo_type = "msa", 
+                      geo_values = name_to_cbsa(c("Pittsburgh", "New York", 
+                                                  "San Antonio", "Miami")))
+
+plot(dv, plot_type = "line", 
+     title = "% of doctor's visits due to COVID-like illness")
+```
+
+# Example: Symptoms
+
+How do my county and my friend's county compare in terms of people reporting that they know somebody with COVID symptoms?
+
+```{r, fig.width=7, fig.height=5}
+sympt = covidcast_signal(data_source = "fb-survey", 
+                         signal = "smoothed_hh_cmnty_cli",
+                         start_day = "2020-04-15", 
+                         geo_values = c(name_to_fips("Allegheny"),
+                                        name_to_fips("Fulton", state = "GA")))
+
+plot(sympt, plot_type = "line", range = range(sympt$value),
+     title = "% of people who know somebody with COVID symptoms")
+```
+
+# API Specification
+
+The COVIDcast API is based on HTTP GET queries and returns data in JSON form. The base URL is https://delphi.cmu.edu/epidata/api.php?source=covidcast
```

Comment:
We probably want to advertise the base URL as https://api.covidcast.cmu.edu/epidata/api.php?source=covidcast (that's the one that gets sent to our AWS cluster and can handle the traffic)

Reply:
Thanks, fixed!